### ISSUE TITLE
how to catch untranslated object property values

### ISSUE THREAD

**Issue by ryancwalsh** [edvardchen/eslint-plugin-i18next#131]
Thanks for this library. It looks like it will be super useful.

Currently, my rule in my `.eslintrc.json` is:

```json
"i18next/no-literal-string": [
            "warn",
            {
                "jsx-attributes": {
                    "include": ["alt", "placeholder", "aria-label", "title", "subtitle"]
                },
                "message": "Use i18n translation keys instead of literal strings.",
                "mode": "jsx-only",
                "object-properties": {
                    "include": ["title", "body", "label"]
                }
            }
        ]
```

It's working well in some cases like:

![image](https://github.com/user-attachments/assets/f8c9a788-fd2c-4a20-90ee-c20bff316044)

But I was hoping it would also flag a warning for objects like this:

```ts
const dialogProps: DialogProps = {
  type: 'icon',
  title: 'Card Verification failed',
  body: 'Something went wrong. Please recheck your card details and try again.',
  dismissButton: {
    label: 'Dismiss',
  },
  dismiss: unshift,
};
```

No warnings: 

![image](https://github.com/user-attachments/assets/cddc996d-f33e-457a-8142-9edb13e5bb65)

What changes can I make to get it to catch strings like those?

Thank you so much for your help!

**Comment by edvardchen** [comment]
Use `mode: 'all'` instead. You might be surprised what it captures
